.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to MsQuic
+
+We'd love your help with MsQuic! Here are our contribution guidelines.
+
+- [Code of Conduct](#code-of-conduct)
+- [Bugs](#bugs)
+- [New Features](#new-features)
+- [Contributor License Agreement](#contributor-license-agreement)
+- [Contributing Code](#contributing-code)
+  - [Process](#process)
+  - [Tests](#tests)
+
+> **Important** - We are still bringing up important regression tests for the core code. Until they are onboarded, any external contributions to the [core](../src/core) or kernel mode files in the [platform](../src/platform) will not be accepted. This is only a **temporary restriction** and will hopefully be removed in the coming months.
+
+## Code of Conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Microsoft Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with additional questions or comments.
+
+## Bugs
+
+One of the easiest ways to contribute is to participate in discussion on GitHub issues.
+
+If you find a non-security related bug, you can help us by [submitting a GitHub Issue](https://github.com/microsoft/msquic/issues/new/choose). The best bug reports provide a detailed description of the issue and step-by-step instructions for predictably reproducing the issue. Even better, you can submit a Pull Request with a fix.
```

Comment:
I'd go with "reliably" instead of "predictably".

Reply:
```suggestion
If you find a non-security related bug, you can help us by [submitting a GitHub Issue](https://github.com/microsoft/msquic/issues/new/choose). The best bug reports provide a detailed description of the issue and step-by-step instructions for reliably reproducing the issue. Even better, you can submit a Pull Request with a fix.
```